docs/source/releases.rst

```diff
@@ -2,6 +2,59 @@ Release History
 ###############
 
 
+v2.0.0 (2022-10-20)
+===================
+
+API Changes
+-----------
+- Removed deprecated ``happi.containers.Device`` container.
+- Removed deprecated methods:
+    * ``happi.Client.create_device``
+    * ``happi.Client.add_device``
+    * ``happi.Client.find_device``
+    * ``happi.Client.all_devices``
+    * ``happi.Client.remove_device``
+    * ``happi.SearchResult.device``
+
+Features
+--------
+- Added ``EntryInfo`` keyword argument ``include_default_as_kwarg``.  If set to ``False``,
+  any keys that are included in an item's ``kwargs`` that match the default of their
+  corresponding ``EntryInfo`` will be omitted from the keyword arguments passed to
+  ``device_class`` when instantiating (loading) the item as in ``happi.loader.load_device`` or
+  ``SearchResult.get()``.
+  If the ``kwargs`` EntryInfo sets ``include_default_as_kwarg = True``,
+  the setting on the corresponding ``EntryInfo`` will be used to decide
+  whether or not to omit a keyword argument.
+  If the ``kwargs`` EntryInfo sets ``include_default_as_kwarg = False``,
+  the setting on corresponding ``EntryInfo`` will be ignored.
+  The default value is True on all EntryInfo instances, retaining the original behavior.
+- For happi load, fall back to Python REPL if IPython is not available.
+- Added MultiBackend, which allows a happi Client to serve information
+  from multiple databases simultaneously.  Updates config parsing logic
+  to match.
+- Added ``happi audit`` function for running checks on happi database items.
+- Restored --json option for ``happi search``.
+
+Bugfixes
+--------
+- Removed extraneous extraneous print in `happi load`.
```

Comment:
Is single quote italic and double code in rst? I get this confused too easily...

```suggestion
- Removed extraneous extraneous print in ``happi load``.
```

Reply:
Refreshing my memory- single quote italic hits the sphinx "default_role" thing, unless that is unset, in which case it does a block.
This repo has a default role set, so you're right.
https://github.com/pcdshub/happi/blob/d822eb50920c126737089bccd7448b91506bebf7/docs/source/conf.py#L99